core/network/adapters/protobuf_block_request.hpp

```diff
@@ -70,15 +73,21 @@ namespace kagome::network {
         } break;
 
         case msg.kNumber: {
-          out.from = std::stoull(msg.number());
+          libp2p::multi::UVarint varint(gsl::make_span(
```

Comment:
Wonder to see something that says it should varint instead of ull. Source or doc